src/snippets/template.pl.tmpl

```diff
@@ -139,30 +139,31 @@ sub main {
         print "\n";
       <% } else if (fileType === "fasta") { %>
       if (defined $item->{"entries"}){
+        print ">$item->{metadata}->{accession}$HEADER_SEPARATOR";
         foreach my $entry (@{$item->{"entries"}}) {
+          print " $entry->{accession}(";
```

Comment:
The leading space before the entry accession could be removed if there is one entry only. And if there is more than one, it could be replace with another separator.